pendingchanges/fixseampoint.txt

```diff
@@ -0,0 +1,2 @@
+Bug Fixes:
+- Fix uninitialized point flags in tools::VolumeToMesh
```

Comment:
Did you get a compiler warning with this or have you seen UB in prod? We should probably do a patch release if so

Reply:
UB in prod.  The Convert VDB SOP randomly initializes seampoints and showed up in our reg tests.

A patch is probably a good idea, maybe wait a bit to see if any other shows up?

Reply:
Eh, it's clearly UB and if the general case exhibits it I would advocate for a `10.0.1` release now.